xds/src/main/java/io/grpc/xds/WeightedRoundRobinLoadBalancer.java

```diff
@@ -340,111 +328,108 @@ public boolean isEquivalentTo(RoundRobinPicker picker) {
     }
   }
 
-  /**
-   * The earliest deadline first implementation in which each object is
-   * chosen deterministically and periodically with frequency proportional to its weight.
-   *
-   * <p>Specifically, each object added to chooser is given a deadline equal to the multiplicative
-   * inverse of its weight. The place of each object in its deadline is tracked, and each call to
-   * choose returns the object with the least remaining time in its deadline.
-   * (Ties are broken by the order in which the children were added to the chooser.) The deadline
-   * advances by the multiplicative inverse of the object's weight.
-   * For example, if items A and B are added with weights 0.5 and 0.2, successive chooses return:
+  /*
+   * The Static Stride Scheduler is an implementation of an earliest deadline first (EDF) scheduler
+   * in which each object is chosen periodically with frequency proportional to its weight.
+   * <p>
+   * Specifically, each backend is given a deadline equal to the multiplicative inverse of 
+   * its weight. The place of each backend in its deadline is tracked, and each call to
```

Comment:
This (L335-L337) is describing the priority queue implementation of EDF and is no longer the case here, but sounds like it is how current implementation is.